client/reference/android/api-reference.mdx

```diff
@@ -1,4 +1,4 @@
 ---
 title: "API Reference"
-url: "https://docs-android.rtvi.ai/rtvi-client-android/index.html"
```

Comment:
After the rename these directories will be different, so for now just point to the root.